<launch>
    <!-- <arg name="video_device" default="/dev/video3"/> -->
    <arg name="camera" default="realsense"/>
    <!-- <arg name="viz" default="true"/> -->
    <!-- <arg name="width" default="1280"/> -->
    <!-- <arg name="height" default="720"/> -->
    <arg name="rviz_config" default="real_sense" />


    <!--    <group ns="$(arg camera)">-->
    <!-- <node name="$(arg camera)" pkg="usb_cam" type="usb_cam_node" output="screen"> -->
        <!-- <param name="video_device" value="$(arg video_device)"/> -->
        <!-- <param name="camera_name" value="$(arg camera)"/> -->
        <!-- <param name="image_width" value="$(arg width)"/> -->
        <!-- <param name="image_height" value="$(arg height)"/> -->
        <!-- <param name="pixel_format" value="mjpeg"/> -->
        <!-- <param name="camera_frame_id" value="$(arg camera)_optical_frame"/> -->
        <!-- <param name="io_method" value="mmap"/> -->
    <!-- </node> -->

    <node name="$(arg camera)" pkg="openni2_camera" type="openni2_camera_node" output="screen" />

    <!-- <group if="$(arg viz)"> -->
        <!-- <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen"> -->
            <!-- <remap from="image" to="/$(arg camera)/rgb/image_raw"/> -->
            <!-- <param name="autosize" value="true"/> -->
        <!-- </node> -->
    <!-- </group> -->
    
    <node name="rviz" pkg="rviz" type="rviz" 
        args="--display-config $(find zau_bringup)/config/$(arg rviz_config).rviz" required="false" >
    </node>
    
</launch>
